app/src/main/java/org/lsposed/manager/ui/fragment/SettingsFragment.java

```diff
@@ -167,6 +199,15 @@ public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
                 prefVerboseLogs.setOnPreferenceChangeListener((preference, newValue) -> ConfigManager.setVerboseLogEnabled(!(boolean) newValue));
             }
 
+            Preference saveLogs = findPreference("save_logs");
+            if (saveLogs != null) {
+                saveLogs.setEnabled(!BuildConfig.DEBUG && installed);
```

Comment:
For module developers usually don't need to care about the build variant of LSPosed, they just need to be able to export the module's logs.

```suggestion
                saveLogs.setEnabled(installed);
```